src/Castle.Windsor/Windsor/Extensions/WindsorContainerExtensions.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2004-2011 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Windsor.Extensions
```

Comment:
@jnm2 I just realised these Resolve extension methods are in `Castle.Windsor.Extensions`, I'm not sure I like that fact you have to drag in yet another namespace every time you want to use them. I think we should move this to the `Castle.Windsor` namespace, otherwise no one is going to discover them.

Reply:
Changed namespace for extension methods.